ethcore/src/spec/spec.rs

```diff
@@ -135,6 +135,12 @@ impl From<ethjson::spec::Spec> for Spec {
 	}
 }
 
+macro_rules! load_bundled {
+	($e:expr) => {
+		Spec::load(include_bytes!(concat!("../../res/", $e, ".json")) as &[u8]).expect("Chain spec is invalid")
```

Comment:
the error messages here might get a little worse than before. the `concat!` macro can customize the `expect` message as well.